stock_picking_report_internal_delivery_address/tests/test_stock_picking_report_internal_delivery_address.py

```diff
@@ -0,0 +1,46 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestStockPicking(TransactionCase):
```

Comment:
This test, for being effective of the final goal of this module, should print the delivery slip, and check that the delivery address is included in it. You can get it as HTML like in https://github.com/OCA/account-invoice-reporting/blob/4e5c239911f49535b0a5506c2d504a07a76227d1/account_invoice_report_grouped_by_picking/tests/test_account_invoice_group_picking.py#L127, and then directly look for the text in the string (instead of parsing it with `lxml` and look for the node, that seems overkill for this case if you put a very especial delivery address to make sure there's no other match in the whole HTML text).